packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2035,6 +2046,14 @@ class InputDecoration {
   /// See [Icon], [ImageIcon].
   final Widget prefixIcon;
 
+  /// Optional widget to place on the line before the input.
+  /// Can be for example used to add some padding to the [prefixText] or to
+  /// add a custom widget in front of the input.
+  ///
+  /// Either [prefix] or [prefixText] can be declared.
```

Comment:
Only one of [prefix] and [prefixText] can be specified.

You can leave the "Declaring both" statement out.